lib/tools/src/cover.erl

```diff
@@ -2525,11 +2523,64 @@ fill1() ->      "..|  ".
 fill2() ->       ".|  ".
 fill3() ->        "|  ".
 
+%% HTML sections
+create_header(OutFile, Enc) ->
+    ["<!doctype html>\n"
+    "<html>\n"
+    "<head>\n"
+    "<meta charset=\"",html_encoding(Enc),"\">\n"
+    "<title>",OutFile,"</title>\n"
+    "<style>"] ++
+    read_stylesheet() ++
+    ["</style>\n",
+    "</head>\n"
+    "<body>\n"
+    "<h1><code>",OutFile,"</code></h1>\n"].
+
+create_footer(ErlFile, Timestamp) ->
+    ["<footer>File generated from <code>",ErlFile,
+    "</code> by <a href=\"http://erlang.org/doc/man/cover.html\">cover</a> at ",
+    Timestamp,"</footer>\n<table>\n<tbody>\n"].
+
+close_html() ->
+    lists:concat(["</tbody>\n",
```

Comment:
You probably don't need `lists:concat/1` here. Returning a list of lists should be fine!

Reply:
@josevalim Thanks for the review!